README.md

```diff
@@ -179,10 +179,8 @@ Refer to the example config found in [test_variables.env.example](test_variables
 To select the network, please set the `NETWORK_TEST_NETWORK_NAME` environment variable. Currenty, the allowed options are:
   - `SEPOLIA_TESTNET`
   - `SEPOLIA_INTEGRATION`
-  - `GOERLI_TESTNET`
-  - `GOERLI_INTEGRATION`
 
-Please note that `GOERLI` networks are deprecated, and won't be supported in the future. The number of tests working on `SEPOLIA` is, however, temporarily limited.
+Note: Some tests are not available following the transition of network tests from `GOERLI` to `SEPOLIA` networks.
```

Comment:
is this going to be addressed by https://github.com/software-mansion/starknet-jvm/pull/466 or you mean something else? this is not very descriptive tbh

Reply:
I think it's something that's gonna be addressed by #384